### DISCUSSION TITLE
Have not got achievment in collaborated repository !!

### DISCUSSION THREAD

**Discussion by dmahdipour** [community/community#58185] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi there,

I have collaborated in a repository that my colleague owned. 
We work hard on it and get about 50 stars. Why did not I get "Starstruck" achievement like him? I have commits and whatever he has done on that repository. why?

**Comment by radar-blip** [comment]
I'm pretty sure only the creator of the repository gets the achievement badge.

**Comment by the-officialjosh** [comment, marked as answer]
So sorry about that, but in order to be recognized as co-authors and receive the badge, both of you need to be added as co-authors at the time of the repository's initial creation. Even if you transferred the repository later on, you would still receive the badge. If you both have a close relationship, one option you could consider is transferring ownership of the repository, as the Starstruck badge would be awarded to both of you at that point. However, I'm uncertain if this approach would work at this time. Nevertheless, it may be worth giving it a try.
